lib/data/licence_transaction/common_licence_identifiers.txt

```diff
@@ -1,228 +0,0 @@
-1245-7-1
```

Comment:
I think deleting these files are good. I think all of the removed files could have been deleted in a separate commit to make this one easier to review.

Reply:
I've moved them to a separate commit 👍